backend/src/main/java/com/bikes/backend/controller/CsrfController.java

```diff
@@ -0,0 +1,14 @@
+package com.bikes.backend.controller;
+
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("api/csrf")
+public class CsrfController {
+    @GetMapping("/")
```

Comment:
"/" is unnecessary. spring is doing it by himself
```suggestion
    @GetMapping
```